app/src/main/res/values/styles.xml

```diff
@@ -59,12 +59,19 @@
     </style>
 
     <style name="MaterialDatePickerTheme" parent="Theme.AppCompat.Light.Dialog">
-        <item name="colorAccent">@color/accent</item>
+        <item name="colorAccent">@color/primary</item>
     </style>
     
     <!--Theme for Spash-->
     <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
         <item name="android:windowBackground">@drawable/splash_background</item>
     </style>
 
+    <style name="FloatingActionButton" parent="Widget.MaterialComponents.FloatingActionButton">
```

Comment:
can we use a better name for the `style`, is same as the view. which creates confusion

Reply:
Can i rename it to `primaryColorFAB`?

Reply:
`FloatingActionButtonTheme` should do. which intuitively means Default theme for FAB

Reply:
@miPlodder Done👍